cpu_scheduling_algorithms/Least_Recently_Used_scheduling.cpp

```diff
@@ -0,0 +1,65 @@
+#include <iostream>
```

Comment:
please document the code according to the contributing guidelines present in the root directory